documentation/en/user/source/configuration/storage.rst

```diff
@@ -272,6 +271,173 @@ GeoServer ``topp:states`` sample layer on a fictitious ``my-geowebcache-bucket``
+
+File Path Templates
+````````````````````
+
+An MBTiles file will correspond to an SQLitle database file. In order to limit the amount of contention on each single database file we will allow users to decide the granularity of the databases files. When GeoWebCache needs to map a tile to a database file it will only look at the databases files paths, it will not take in account the MBTiles metadata (this is why we are able to handle others formats and projections).
+
+To configure the databases files granularity the user needs to provide a file path template. The default file path template for the MBTiles blob store is this one:
+
+.. code-block:: none
+
+  {grid}\{layer}\{format}\{params}\{z}\tiles_{x}_{y}.sqlite
+
+The provided files paths templates will always be considered relative to the root directory provided as a configuration option.
+
+In this case we are storing our tiles organized by the grid set id, the layer name, the tiles format, the parameters unique hash, the zoom level, column range and row range. The column and row range values are passed by configuration, by default those values are equal to 250.
+
+Follows an example of what the blob store root directory structure may look like when using the default file path template:
+
+.. code-block:: none
+
+  .
+  `-- EPSG_4326
+      `-- sf_restricted
+          `-- image_png
+              |-- f0023dc7bc347fee7a3a04dc797f2223f74e3448
+              |   `-- 14
+              |       `-- tiles_6750_12000.sqlite
+              `-- null
+                  |-- 10
+                  |   |-- tiles_0_0.sqlite
+                  |   |-- tiles_250_500.sqlite
+                  |   `-- tiles_250_750.sqlite
+                  `-- 9
+                      |-- tiles_0_0.sqlite
+                      `-- tiles_0_250.sqlite
+
+If no parameters were provided *null* string will be used. Is the responsibility of the user to define a file path template that will avoid collisions.
+
+The terms that can be used in the file path template are:
+
+* **grid**: the grid set id
+* **layer**: the name of the layer
+* **format**: the image format of the tiles
+* **params**: parameters unique hash
+* **x**: column range, computed based on the column range count configuration property
+* **y**: row range, computed based on the row range count configuration property
+* **z**: the zoom level
+
+It is also possible to use parameters values, like *style* for example. If the parameter is not present *null* will be used.
+
+.. note:: The file template separators are specific to the operating system used, for example ``\`` for Linux and ``/`` for Windows. Any special char like \, /, : or empty space used in a term value will be substituted with an underscore.
```

Comment:
This means the configuration cannot be moved from a developer laptop on Windows to a Linux server without changes?